common/lib/xmodule/xmodule/tests/test_import.py

```diff
@@ -208,7 +210,8 @@ def course_descriptor_inheritance_check(self, descriptor, from_date_string, unic
         )
 
         # Now export and check things
-        descriptor.runtime.export_fs = MemoryFS()
+        file_system = OSFS(mkdtemp())
+        descriptor.runtime.export_fs = file_system.makedir(u'course', recreate=True)
```

Comment:
where are we using `export_fs`?

Reply:
This was already used, We just changed to OSFS in tests so as to be consistent with actual implementation.